contracts/utils/MultiRateLimited.sol

```diff
@@ -0,0 +1,270 @@
+
+        maxRateLimitPerSecond = _maxRateLimitPerSecondSubGovernor;
+        maxBufferCap = _maxBufferCapSubGovernor;
+    }
+
+    // ----------- Governor and Admin only state changing api -----------
+
+    /// @notice update the sub gov max rate limit per second
+    /// @param newMaxRateLimitPerSecond new maximum rate limit per second for sub governors
+    /// TODO determine the proper modifier for this function
+    function updateSubGovMaxRateLimitPerSecond(uint256 newMaxRateLimitPerSecond) external virtual override onlyGovernorOrAdmin {
+        uint256 oldMaxRateLimitPerSecond = maxRateLimitPerSecond;
+        maxRateLimitPerSecond = newMaxRateLimitPerSecond;
+
+        emit SubGovMaxRateLimitPerSecondUpdate(oldMaxRateLimitPerSecond, newMaxRateLimitPerSecond);
+    }
+
+    /// @notice update the sub gov max buffer cap
+    /// @param newBufferCap new buffer cap for sub governors
+    /// TODO determine the proper modifier for this function
+    function updateSubGovMaxBufferCap(uint256 newBufferCap) external virtual override onlyGovernorOrAdmin {
+        uint256 oldBufferCap = newBufferCap;
+        maxBufferCap = newBufferCap;
+
+        emit SubGovBufferCapUpdate(oldBufferCap, newBufferCap);
+    }
+
+    /// @notice add an authorized rateLimitedAddress contract
+    /// @param rateLimitedAddress the new address to add as a rateLimitedAddress
+    /// @param _rateLimitPerSecond the rate limit per second for this rateLimitedAddress
+    /// @param _bufferCap  the buffer cap for this rateLimitedAddress
+    function addAddress(address rateLimitedAddress, uint112 _rateLimitPerSecond, uint144 _bufferCap) external virtual override onlyGovernorOrAdmin {
+        _addAddress(rateLimitedAddress, _rateLimitPerSecond, _bufferCap);
+    }
+
+    /// @notice add an authorized rateLimitedAddress contract
+    /// @param rateLimitedAddress the address whose buffer and rate limit per second will be set
+    /// @param _rateLimitPerSecond the new rate limit per second for this rateLimitedAddress
+    /// @param _bufferCap  the new buffer cap for this rateLimitedAddress
+    function updateAddress(address rateLimitedAddress, uint112 _rateLimitPerSecond, uint144 _bufferCap) external virtual override onlyGovernorOrAdmin {
+        require(_bufferCap <= bufferCap, "MultiRateLimited: buffercap too high");
+
+        _updateAddress(rateLimitedAddress, _rateLimitPerSecond, _bufferCap);
+    }
+
+    /// @notice add an authorized rateLimitedAddress contract
+    /// @param rateLimitedAddress the new address to add as a rateLimitedAddress
+    /// @param _rateLimitPerSecond the rate limit per second for this rateLimitedAddress
+    /// @param _bufferCap  the buffer cap for this rateLimitedAddress
+    /// TODO figure out what the modifier should be for this function
+    function addAddressSubGovernor(address rateLimitedAddress, uint112 _rateLimitPerSecond, uint144 _bufferCap) external virtual override onlyGovernorOrAdmin {
```

Comment:
remove the rate limit per second and buffer cap and automatically set all calls from sub gov to these global values

use the new roles

remove the admin modifiers from this contract